clients/src/main/java/org/apache/kafka/common/metrics/QuotaViolationException.java

```diff
@@ -51,4 +47,20 @@ public double value() {
     public double bound() {
         return bound;
     }
+
+    @Override
+    public String toString() {
+        return String.format(
```

Comment:
Do we need `String.format` here? Seems like String concat would be fine.

Reply:
Sounds good. I figured it wasn't used so simply moved it but seems better to be safe.